samples/core/Modeling/TableSplitting/DetailedOrder.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace TableSplitting
+{
+    #region DetailedOrder
+    public class DetailedOrder : Order
```

Comment:
DetailedOrder inherits a DetailedOrder property from Order. I wonder what happens with it.

Reply:
Good point, I'll ignore it for clarity

Reply:
If the reason for using inheritance is to avoid repeating the properties, wouldn't a common base type (i.e. OrderData or OrderBase) that only has ID and status and isn't included in the model be a bit cleaner?

Reply:
There hasn't been an opportunity to use `HasBaseType((string)null)` in samples yet and I wanted to show that it's possible

Reply:
Okay